src/sources/languages.ts

```diff
@@ -9,7 +11,11 @@ export default function getLanguages(): string[][] {
   }
 
   if (Array.isArray(n.languages)) {
-    result.push(n.languages)
+    // Starting from Chromium 86, there is only a single value in `navigator.language` in Incognito mode:
+    // the value of `navigator.language`. Wherefore the value is ignored in this browser.
+    if (!isChromium() || !isChromium86OrNewer()) {
```

Comment:
can we invert this condition please? It will be easier to understand if we handle this case in an assertive case (if chrome && newChrome){.

Reply:
I agree, done